README.md

```diff
@@ -26,22 +26,13 @@ The fields in the table below can be used in these parts of fiboa documents:
 
 | Property Name   | Type   | Description |
 | --------------- | ------ | ----------- |
-| ml:creation_method | string | **REQUIRED**. how the label was created e.g. field walk, image-annotated, ML generated |
-| ml:creation_method_detail | string |  additional details about the creation method, including potentially a link to the algorithm |
 | ml:boundary_type | string | **REQUIRED**. whether the boundary is for a field, ownership parcel, hedge, or other boundary |
 | ml:creation_date | datetime  | **REQUIRED**.[Auto-added] Date collection was created or published |
 | ml:verified_from | datetime | **REQUIRED**. the first date for which the boundary is verified to be present on ground (e.g. through ground-truth or remote sensing image verification) |
```

Comment:
The data type says datetime, but the description only date. What is meant here?

Reply:
It was supposed to be date – what's the right format for JSON? is it `date` or `Date`?

Reply:
JSON is irrelevant here, we use fiboa Schema data types:
https://github.com/fiboa/schema/blob/main/datatypes.md

So it's `date` for day, month, year (JSON encoding: "YYYY-MM-DD") and `datetime` for anything that's more fine-granular.

You actually may also want to watch the tutorial from today:
https://github.com/fiboa/tutorials/tree/main/create-extension

Reply:
Got it, thank you!